website/docs/wallet/withdraw-validator.md

```diff
@@ -16,87 +18,68 @@ import TabItem from '@theme/TabItem';
 
 :::caution Public Preview
 **This feature is currently in public preview** and may change significantly as we receive feedback from users like you. Join our [Discord server](https://discord.gg/prysmaticlabs) to share your feedback.
-Withdrawals are not yet available as a feature on Ethereum mainnet, and are currently only available on development test networks for trial at the time of writing (February 1st, 2023). The instructions in this document WILL NOT work on mainnet and are subject to change by the time withdrawals are available on mainnet.
+
+Note that withdrawals **aren't yet available on Ethereum mainnet**; this functionality is currently being validated on test networks. The instructions in this document won't work on Ethereum mainnet and may change significantly as the feature is stabilized on test networks.
 :::
 
-## Background
 <img  src={CapellaWithdrawalPng} /> 
-In the **Capella/Shanghai Ethereum** upgrade, stakers will be able to withdraw their validator(s)’ funds from staking on the network. When withdrawing, stakers can either choose to 
 
-(a) **partially withdraw** their **earnings**, that is, all value staked above 32 ETH, and continue validating
+The **Capella/Shanghai Ethereum** upgrade lets you withdraw your validator nodes' staked Ethereum in one of two ways: 
+
+ 1. **Partial (earnings) withdrawal**: This option lets you withdaw your earnings (that is, all value staked above 32 ETH) and continue validating.
+ 2. **Full withdrawal**: This option lets you liquidate your entire stake and earnings, effectively liquidating your validator node(s) and exiting the network.
+
+In this how-to, you'll learn how to perform both types of withdrawals.
 
-OR 
 
-(b) **fully withdraw** their validator(s), which involves exiting the network and no longer staking, but gaining custody of the validator(s)’ full balances
 
-This document provides a comprehensive guide for users running Prysm to perform either of the two withdrawal actions, and includes a series of tips, warnings, and frequently asked questions.
 
-Beyond these docs for the Prysm client, a comprehensive FAQ has been published [here](https://notes.ethereum.org/@launchpad/withdrawals-faq) which covers all key things to note regarding how validator withdrawals work on Ethereum.
+## Before you begin
```

Comment:
Bit more consolidation and copy reduction here, purely structural changes made in the spirit of streamlining the content experience.